espnet/nets/chainer_backend/asr_interface.py

```diff
@@ -0,0 +1,22 @@
+"""ASR Interface module."""
+
+from espnet.nets.asr_interface import ASRInterface
+
+
+class ChainerASRInterface(ASRInterface):
```

Comment:
Maybe this is a little bit better?
```suggestion
class ChainerASRInterface(ASRInterface, chainer.Chain):
```